.github/pull_request_template.md

```diff
@@ -1,36 +1,28 @@
 Fixes | Closes | Resolves #
 
-> Anything marked as optional that you didn't need to fill in, please remove it from the PR description. Choose one of the keywords above to refer to the issue this PR solves, followed by the issue number (e.g Fixes # 666). If there is no issue, remove the line. Remove this note after reading.
-## Changes:
--
-
-## Types of changes
-
-What types of changes does your code introduce?
-_Put an `x` in the boxes that apply_
+## Description
 
-- [ ] Bugfix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] Documentation Update
-- [ ] Code style update (formatting, renaming)
-- [ ] Refactoring (no functional changes, no api changes)
-- [ ] Build related changes
-- [ ] Other (please describe): 
+_Explanation about why the PR is necessary_
 
-## Testing
-**Requires testing**
+## Changes:
 
-- [ ] Yes
-- [ ] No
+- Change 1
+- Change 2
 
-**In case you checked yes, did you write tests??**
+## Types of changes
 
-- [ ] Yes
-- [ ] No
+_Leave only the ones that describe your changes, remove the rest. Can remove this line too_
 
-**Comments about testing , should you have some** (optional)
+- Bugfix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- Documentation Update
+- Code style update (formatting, renaming)
+- Refactoring (no functional changes, no api changes)
+- Build related changes
+- Other (please describe): 
 
-## Further comments (optional)
+## Testing
+**Requires testing**: Yes/No
+**Did you write tests??**: Yes/No
```

Comment:
This renders as a single line. Perhaps a space in between will fix it?

Reply:
Done -> https://github.com/NethermindEth/juno/pull/331/commits/3c9da69405a173a192731bcbcafb856b8defd659